modules/ROOT/pages/monitor-connectors.adoc

```diff
@@ -9,11 +9,12 @@ You also need the Unified Agent.
 
 == Supported Mule Versions
 
-To monitor connectors, you must use a supported version of Mule:
+To monitor connectors, you must use a xref:release-notes::mule-runtime/mule-esb.adoc[supported Mule version] for your apps: 
```

Comment:
points here: https://docs.mulesoft.com/release-notes/mule-runtime/mule-esb
PM good with this